### PR TITLE
TheClient::getAccountsBalances method added

### PR DESCRIPTION
**Pull request by Triplkrypl**
https://redmine.havelholding.cz/issues/4697

### CODE REVIEW

File: tests/GetAccountsBalancesTest.php [hunk 1]
```diff
@@ -0,0 +1,56 @@
+<?php
+
```
Comment (joseftraxler) [inline tests/GetAccountsBalancesTest.php:2]: + strict types

Comment (joseftraxler) [inline tests/GetAccountsBalancesTest.php:2]: \+ strict types?

Comment (Triplkrypl) [reply tests/GetAccountsBalancesTest.php:2]: In version v1.x we still have `~5.3` in composer.json, i am not sure but static analysis can have problem with strict type on old versions.

I can add strict type while i will merging v1.x into v2.x.

Comment (joseftraxler) [reply tests/GetAccountsBalancesTest.php:2]: That's right, but tests run from PHP 7.2 (I don't know, if this version supports strict types). It's not so much important, so it's up to you.

**Review by joseftraxler — APPROVED** [review]
Nice work! 👏

**Pull request merged**